addon/templates/components/flexberry-maplayer.hbs

```diff
@@ -41,6 +43,12 @@
       <div style="clear: both;"></div>
     {{/if}}
 
+    {{#if _canChangeOpacity}}
+      <label class="{{flexberryClassNames.opacityLabel}}" title="{{t "components.flexberry-maplayer.opacity" }}">
```

Comment:
To user it looks similar to buttons, click on it causes expand/collapse animation on related tree-node (because css-class flexberryClassNames.preventExpandCollapse isn't applied), so usability is strange.
It is better to display current opacity somewhere near the slider (above or under it, as a tooltip maybe), isn't it?